uber/models.py

```diff
@@ -1048,7 +1049,10 @@ def _status_adjustments(self):
     @presave_adjustment
     def _staffing_adjustments(self):
         if self.ribbon == c.DEPT_HEAD_RIBBON:
-            self.staffing = self.trusted = True
+            self.staffing = True
+            # question: do we have info about what this person is a dept head of? or is that not something
+            # we know explicitly?  if so, we can either make them trusted in every dept, or trusted in no departments.
+            # self.trusted_depts = self.assigned_depts # this might be a reasonable thing to do. or not.
```

Comment:
I'd be inclined to err on the side of making dept heads trusted in all of the departments they're assigned to.  I'm not that concerned about a department head taking a shift they're "not supposed to" in another department, whereas I am concerned that people will not see shifts that they ought to see.